WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PhaseStepping/StepOutputRegistrationTests.cs

```diff
@@ -33,17 +33,19 @@ public async Task AllBobsRegisteredAsync()
 
 			// Register Alices.
 			using Key key1 = new();
-			var irreq1 = WabiSabiFactory.CreateInputsRegistrationRequest(key1, round);
+			var irreq1 = WabiSabiFactory.CreateInputRegistrationRequest(key1, round);
 			var irres1 = await arena.RegisterInputAsync(
 				irreq1.RoundId,
-				irreq1.InputRoundSignaturePairs.ToDictionary(x => new Coin(x.Input, new TxOut(Money.Coins(1), key1.PubKey.GetSegwitAddress(Network.Main))), x => x.RoundSignature),
+				new Coin(irreq1.Input, new TxOut(Money.Coins(1), key1.PubKey.GetSegwitAddress(Network.Main))),
+irreq1.RoundSignature,
 				irreq1.ZeroAmountCredentialRequests,
 				irreq1.ZeroVsizeCredentialRequests);
```

Comment:
this boilerplate is repeated a lot, `CreateCoin` can be used to clean it up a tiny bit, but I didn't do that yet because it seems we can do better by refactoring `WabiSabiFactory` some more